arch/x86_64/fentry.S

```diff
@@ -1,8 +1,44 @@
 /* argument passing: %rdi, %rsi, %rdx, %rcx, %r8, %r9 */
 /* return value: %rax */
 /* callee saved: %rbx, %rbp, %rsp, %r12-r15 */
-/* stack frame (with -pg): return addr = 8(%rbp), prev fp = 0(%rbp) */
-/* stack frame (with -fentry): return addr = (%rsp), prev fp = 8(%rsp) */
+/* stack frame (with -pg -mfentry): parent location = addr of 52(%rsp), child ip = 48(%rsp) */
```

Comment:
52 seems wrong.  And I'd like to see stack frame info at the beginning of mcount() - not after libmcount.so:mcount subtracts the stack pointer.  Also please just show parent addr and child addr like "parent addr = 8(%rsp), child addr = (%rsp)"